src/Ip.php

```diff
@@ -74,6 +74,37 @@ public static function isValidv6($ip)
     }
 
     /**
+     * Checks if an IP is local
+     *
+     * @param  string  $ip IP
+     * @return boolean     true if the IP is local, otherwise false
+     */
+    public static function isLocal($ip)
+    {
+        if (self::isValidv4($ip)) {
+            return self::match($ip, '10.*.*.*') ||
+                   self::match($ip, '127.*.*.*') ||
+                   self::match($ip, '192.168.*.*') ||
+                   self::match($ip, '169.254.*.*') ||
+                   self::match($ip, '172.16.*.*') ||
```

Comment:
`172.16.*.*` is a little specific it should be `172.16/12` aka `172.16.0.0 - 172.31.255.255` ([reference](https://tools.ietf.org/html/rfc1918#page-4))